### PR TITLE
My Amazing Commit

### PR DESCRIPTION
**Pull request by ophillipsBrandwatch**

### CODE REVIEW

**Review by SamJamesKennedy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Faffew is really awesome.
+Faffew is really awesome and so is Olivia.
```
Comment (bradleykhan) [inline README.md:1]: This should be and so is 'Bradley'

**Review by bradleykhan — CHANGES_REQUESTED** [review]

**Review by richardbadman — APPROVED** [review]

**Review by SergeEgorov — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chaicesan) [reply README.md:1]: I prefer new lines at teh end of files

**Review by olga-brandwatch — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ophillipsBrandwatch) [reply README.md:1]: 👎

**Comment by bw-matthew** [comment]
This PR wins the review war with 4 approvals! Well done :smile:

**Comment by richardbadman** [comment]
Can I retract my approval?

**Pull request merged**